src/bundle/Command/MigrateNamespacesCommand.php

```diff
@@ -0,0 +1,144 @@
+
+        // Check that both --cursor-start and cursor-start are set, or neither
+        if (($this->cursorStart === null) xor ($this->cursorStop === null)) {
+            throw new RuntimeException('The options --cursor-start and -cursor-stop are only for internal use !');
+        }
+
+        parent::execute($input, $output);
+
+        return self::SUCCESS;
+    }
+
+    protected function getObjectCount(): int
+    {
+        return $this->gateway->countRichtextAttributes();
+    }
+
+    protected function iterate(): void
+    {
+        $limit = $this->getIterationCount();
+        $cursor = [
+            'start' => -1,
+            'stop' => null,
+        ];
+
+        $contentAttributeIDs = $this->gateway->getContentObjectAttributeIds($cursor['start'], $limit);
+        $cursor['stop'] = $this->getNextCursor($contentAttributeIDs);
+        while ($cursor['stop'] !== null) {
+            $this->createChildProcess($cursor, count($contentAttributeIDs));
+
+            $cursor['start'] = $cursor['stop'];
+            $contentAttributeIDs = $this->gateway->getContentObjectAttributeIds($cursor['start'], $limit);
+            $cursor['stop'] = $this->getNextCursor($contentAttributeIDs);
+        }
+    }
+
+    protected function completed(): void
+    {
+        $this->output->writeln(PHP_EOL . 'Completed');
+    }
+
+    protected function getNextCursor(array $contentAttributeIDs): ?int
+    {
+        $lastId = count($contentAttributeIDs) > 0 ? end($contentAttributeIDs)['id'] : null;
+
+        return $lastId;
+    }
+
+    protected function processData($cursor)
+    {
+        $this->updateNamespacesInColumns($cursor['start'], $cursor['stop']);
+    }
+
+    protected function constructCursorFromInputOptions(): mixed
+    {
+        return [
+            'start' => $this->cursorStart,
+            'stop' => $this->cursorStop,
+        ];
+    }
+
+    protected function addChildProcessArguments($cursor): array
+    {
+        return [
+            '--cursor-start=' . $cursor['start'],
+            '--cursor-stop=' . $cursor['stop'],
+        ];
+    }
+
+    protected function isChildProcess(): bool
+    {
+        return $this->cursorStart !== null || $this->cursorStop !== null;
+    }
+
+    protected function updateNamespacesInColumns(int $contentAttributeIdStart, int $contentAttributeIdStop): void
+    {
+        $contentAttributes = $this->gateway->getContentObjectAttributes($contentAttributeIdStart, $contentAttributeIdStop);
+
+        foreach ($contentAttributes as $contentAttribute) {
+            $contentAttribute['data_text'] = str_replace('xmlns:ezxhtml="http://ez.no/xmlns/ezpublish/docbook/xhtml"', 'xmlns:ezxhtml="http://ibexa.co/xmlns/dxp/docbook/xhtml"', $contentAttribute['data_text']);
+            $contentAttribute['data_text'] = str_replace('xmlns:ezcustom="http://ez.no/xmlns/ezpublish/docbook/custom"', 'xmlns:ezcustom="http://ibexa.co/xmlns/dxp/docbook/custom"', $contentAttribute['data_text']);
```

Comment:
change `ezxhtml` and `ezcustom` to `ezxhtml` and `ezcustom` while we are at it?

That would maybe require some changes in editor and other places too though ?